.github/workflows/ci.yml

```diff
@@ -227,7 +229,6 @@ jobs:
         uses: ansible-community/ansible-test-gh-action@release/v1
         with:
           ansible-core-version: ${{ matrix.ansible }}
-          codecov-token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
It's already an empty string in PRs from forks.

As long as `.codecov.yml` has the token, it's used as a fallback when `${{ secrets.CODECOV_TOKEN }}` is empty.